src/geo.h

```diff
@@ -8,18 +8,91 @@ struct Color
 {
 	Color() : r(0.0f), g(0.0f), b(0.0f), a(false) {};
 	Color(float r, float g, float b) : r(r), g(g), b(b), a(false) {};
-	Color(unsigned r, unsigned g, unsigned b) : a(false)
+	Color(unsigned char r, unsigned char g, unsigned char b) : a(false)
 	{
 		this->r = static_cast<float>(r) / 255.0f;
 		this->g = static_cast<float>(g) / 255.0f;
 		this->b = static_cast<float>(b) / 255.0f;
+		this->rb = r;
```

Comment:
instead of storing redundant data, getters and setters with overloads are prefered